commerce-cloud-code/app_yourshophere/cartridge/partials/plp/grid.js

```diff
@@ -20,11 +20,34 @@ exports.createModel = () => {
     model.moreUrlHx = model.moreUrlHx.append('component', componentId);
     model.desktopColumns = componentSettings.desktopColumns || 3;
     model.mobileColumns = componentSettings.mobileColumns || 2;
+    const analyticsData = {
+        products: search.foundProducts.map((hit) => ({
```

Comment:
Just pulling out the productId is OK - but it sets the wrong pattern we must not propagate here